src/app/models/cake.model.ts

```diff
@@ -0,0 +1,34 @@
+export const enum statusCake {
+  Available = 'Available',
+  LastUnits = 'LastUnits',
+  OutOfStock = 'OutOfStock',
+}
+
+export interface CakeInterface {
```

Comment:
I suggest about the interface name is: `ICake` or `iCake`

Reply:
Cool! Thanks :)